src/components/Input/InputWrapper.tsx

```diff
@@ -253,9 +251,8 @@ export const IconButton = styled.button<{ $show?: boolean }>`
   &:not(:disabled) {
     cursor: pointer;
   }
-  ${({ theme, $show }) => `
+  ${({ theme }) => `
       padding: ${theme.click.field.space.y} 0;
-      visibility: ${$show ? "visible" : "hidden"};
```

Comment:
is there any reason we were hiding icons using visibility rather than not rendering them?